app/common/components/table.test.js

```diff
@@ -70,34 +71,57 @@ describe('table', () => {
     });
   }));
 
-  it('should render <a> for columns with link property', createTestHarness(setup, (harness) => {
+  it('should render <a> for data with link property', createTestHarness(setup, (harness) => {
     const context = { params: { ...mockContext.params } };
-    context.params.columnInfo[0].link = true;
-    context.params.columnInfo[1].link = true;
+    context.params.data[0][0].href = 'organisations/org1';
+    context.params.data[0][1].href = 'ods/X01';
+    context.params.data[1][0].href = 'organisations/org1';
+    context.params.data[1][1].href = 'ods/X02';
     harness.request(context, ($) => {
       context.params.data.forEach((row, rowIndex) => {
         row.forEach((dataPoint, i) => {
-          expect($(`[data-test-id="table-row-${rowIndex}"] a:nth-child(${i + 1})`).text().trim()).toEqual(dataPoint);
-          expect($(`[data-test-id="table-row-${rowIndex}"] a:nth-child(${i + 1})`).hasClass(mockContext.params.columnClass)).toEqual(true);
-          expect($(`[data-test-id="table-row-${rowIndex}"] a:nth-child(${i + 1})`).attr('href')).toEqual('#');
+          const aTag = $(`[data-test-id="table-row-${rowIndex}"] a:nth-child(${i + 1})`);
+          expect(aTag.text().trim()).toEqual(dataPoint.data);
+          expect(aTag.hasClass(mockContext.params.columnClass)).toEqual(true);
+          expect(aTag.attr('href')).toEqual(dataPoint.href);
           expect($(`[data-test-id="table-row-${rowIndex}"] div`).length).toEqual(0);
         });
       });
     });
   }));
 
-  it('should render <div> for columns with link false property', createTestHarness(setup, (harness) => {
+  it('should render <div> for data with no link property', createTestHarness(setup, (harness) => {
     const context = { params: { ...mockContext.params } };
-    context.params.columnInfo[0].link = false;
-    context.params.columnInfo[1].link = false;
+    delete context.params.data[0][0].href;
+    delete context.params.data[0][1].href;
+    delete context.params.data[1][0].href;
+    delete context.params.data[1][1].href;
```

Comment:
Do you need to do this? looks like the mockContext by default doesn't have the `href` property